gp-includes/install-upgrade.php

```diff
@@ -12,9 +12,18 @@
  * @since 1.0.0
  */
 function gp_upgrade_db() {
-	dbDelta( implode( "\n", gp_schema_get() ) );
+	global $wpdb;
 
 	$gp_db_version = get_option( 'gp_db_version' );
+
+	if ( 970 < $gp_db_version ) {
```

Comment:
This should be `$gp_db_version && $gp_db_version < 970`

* It shouldn't run on installation.
* It should run if the db version is **below** 970.